---
layout: page
title: "Data?"
permalink: /data/
tags: Data
---
    <div id="main" class="column grid-13 ">
  	  <div id="content">
  	    <div id="content-inner">

          <div class="breadcrumb"><a href="/">Home</a> › </div>
          
          
                    
          
          <div id="main-content" class="region clear-block">
            
<div id="node-48" class="node node-type-page"><div class="node-inner ">

  
  
  
  
  <div class="content">
    
<p>&nbsp;</p>
<div class="enclosed">
 <h1 class="section_title">1000 Genomes Data and Sample Information</h1>
 <p>The 1000 Genomes Project is a community resource project that aims to release data rapidly for the benefit of the scientific community.</p>
 <a href="#DataAvailable">Description of data released by the project</a><br />
 <a href="#DataAccess">How to Access 1000 Genomes Data</a><br />
 <a href="#DataReleasePolicy">Data Release Policy</a><br />
 <a href="#SampleAvailability">Sample Availability</a><br />
 <a href="#DataUse">Use of the Project data, presentations and publications, and authorship</a>
</div>
<p><a name="DataAvailable" id="DataAvailable"></a></p>
<div class="enclosed">
 <h1 class="section_title">Data released by the 1000 Genomes Project</h1>
 <h2>Sample lists and sequencing progress</h2>
 <p>A summary of sequencing done for each of the three pilot projects is available <a href="/sites/1000genomes.org/files/docs/PilotsSummary.pdf">here</a>. The list of samples and allocations is provided in a <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/working/20130606_sample_info/20130606_sample_info.xlsx" target="_blank">spreadsheet</a>.</p>
 <h2>Variant Calls</h2>
 <p>Our variant calls are always released in <a href="/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">vcf format</a>. The released can be found in the release directory <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release" target="_blank">EBI</a>|<a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release" target="_blank">NCBI</a>.</p>
 <h2>Alignments</h2>
 <p>The main project alignments are available in <a href="http://samtools.sourceforge.net/">BAM</a> format. A list of the files currently available can be found in the alignment index <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/alignment.index">EBI</a>|<a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/alignment.index">NCBI</a>. Alignment statistics can be found in the alignment_indices directory <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/alignment_indices">EBI</a>|<a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/alignment_indices">NCBI</a>. There is also a <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/README.alignment_data">README</a> which explains the alignment process and file layout</p>
 <h2>Raw sequence files</h2>
 <p>The main project raw sequence data is available in fastq format. A list of files currently available can be found in the sequence.index <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/sequence.index">EBI</a>|<a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/sequence.index">NCBI</a> Sequence statistics can be found in the sequence_indices directory <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/sequence_indices">EBI</a>|<a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/sequence_indices">NCBI</a>. There is also a <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/README.sequence_data">README</a> which explains the sequence processing and the file layout</p>
</div>
<p><a name="DataAccess" id="DataAccess"></a></p>
<div class="enclosed">
 <h1 class="section_title">How to Access 1000 Genomes Data</h1>
 <h2>Download data</h2>
 <p>The sequence and alignment data generated by the 1000genomes project is made available as quickly as possible via our mirrored ftp sites.</p>
 <p>EBI FTP: <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/</a><br />
 NCBI FTP: <a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/">ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/</a></p>
 <p>Users in the Americas should use the <a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/">NCBI ftp site</a> and users in Europe and the rest of the world should use the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/">EBI ftp site</a></p>
 <p>The data is also available via an Aspera server from both sites. To be able to use this service you need to download the <a href="http://asperasoft.com/software/transfer-clients/connect-web-browser-plug-in/">Aspera connect</a> software. This provides both a firefox plug in for downloading data and a bulk download client called ascp</p>
 <p>The plugin should automatically start when you visit either the <a href="/aspera">EBI Aspera site</a> or the <a href="http://www.ncbi.nlm.nih.gov/projects/faspftp/1000genomes/">NCBI Aspera site</a>.</p>
 <p>An example command line for ascp looks like:</p>
 <p><em>ascp -i bin/aspera/etc/asperaweb_id_dsa.putty -Tr -Q -l 100M -L- fasp-g1k@fasp.1000genomes.ebi.ac.uk:</em>vol1/ftp/release/20100804/ALL.2of4intersection.20100804.genotypes.vcf.gz <em>./</em></p>
 <p>The argument to -i may be different depending on the location of the default key file. This command should not ask you for a password. All the 1000 genomes data is freely accessible but you do need to give ascp the ssh key to complete the command.</p>
 <h2>FTP Hierachy</h2>
 <p>The FTP site follows a specific data hierachy to enable data discovery</p>
 <p>&nbsp;</p>
 <ul>
  <li>CHANGLOG, This file gives summaries and dates for changes made to the ftp site</li>
  <li>changelog_details, This directory contains file lists specifying particular files which have changed. The filename format is explained in <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/README.ftp_structure" target="_blank">README.FTP_structure</a></li>
  <li>current.tree, This file describes the current file hierachy and the size of all the files in it, what their md5 checksum is and the date they were last upated.</li>
  <li>data, This directory contains another directory per individual named for the sample name like NA12878. The contents of each individual directory is described below in the data directory section</li>
  <li>pilot_data, This directory contains all the data used for the pilot analysis. Its structure mirrors that of the data, release and technical directories of the main ftp site</li>
  <li>release, This directory contains a dated directory for each release of analysis results like SNP or CNV calls. Each directory is named for the sequence index release it is based on in the form YYYYMMDD e.g our first main project release directory is called <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20100804/">20100804</a> to match the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/sequence_indices/">sequence.index</a> is based on.</li>
  <li>technical, This directory contains various technical information required for the project. Its contents is explained further in the technical directory section</li>
 </ul>
 <p>&nbsp;</p>
 <p>The data dir</p>
 <p>Under the data dir there is a directory for every individual sequenced so far as part of the project. Each directory is names with the NAXXXXX identifier assigned to the individual. This system is also mimiced under pilot_data/data</p>
 <p>Each NAXXXX directory contains several directories itself</p>
 <ul>
  <li>sequence_read, This directory contains all the sequence data associated with that individual. This data is also indexed in the <a href="/formats#IndexFiles">sequence.index</a> file.</li>
  <li>alignment, This directory contains all the bam, bai and bas files associated with the alignments for a particular individual. Generally there is one bam file per individual and technology. For high cover individuals each bam file is split on a per chromosome basis. This information is indexd in the <a href="/formats#IndexFiles">alignment.index</a> file.</li>
 </ul>
 <p>&nbsp;</p>
 <p>The technical dir</p>
 <p>Under the technical dir there are several directories designed to store differeent data types. This is primarily a location for exchange of data between project participants. It also stores refence data sets in different directories</p>
 <ul>
  <li>method development, This contains useful files for testing out new methods and assessing other collaborators methods</li>
  <li>qc, This contains information about any qc done on the data in the database</li>
  <li>reference, This contains reference datasets like a copy of the currently used human genome and annotation sets like gene predictions or ancestral alignments</li>
  <li>retired_reference, This contains any reference data which is no longer used by the project</li>
  <li>simulations, This contains any simulated data and the programs used to create it</li>
  <li>working, This contains a series of dated directories which contain technical data required for the ongoing project</li>
 </ul>
 <p>&nbsp;</p>
 <h2>Browse data</h2>
 <p>The first set of SNP calls representing the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/release/2010_07/">July 2010</a> data release can be viewed directly through the 1000 Genomes browser at <a href="http://browser.1000genomes.org" title="http://browser.1000genomes.org">http://browser.1000genomes.org</a>. Launch the browser and <a href="http://browser.1000genomes.org/Homo_sapiens/Location/View?r=6:133017695-133161157" target="_blank">view a sample region here.</a></p>
 <p><strong>For more information about using the 1000 Genomes browser, download the <a href="/sites/1000genomes.org/files/documents/1000genomes_browser_quickstart.pdf">Quick start guide</a></strong></p>
 <h2>Amazon Web Services</h2>
 <p>The alignment files for the 1000 Genomes Pilot Porject are available as a <a href="http://aws.amazon.com/publicdatasets/">public data set</a> via Amazon Web Services (AWS). Further information can be found on 1000 Genomes <a href="http://aws.amazon.com/datasets/4383">public data set page</a> or directly on <a href="http://s3.amazonaws.com/1000genomes">http://s3.amazonaws.com/1000genomes</a>.</p>
</div>
<p><a name="DataReleasePolicy" id="DataReleasePolicy"></a></p>
<div class="enclosed">
 <h1 class="section_title">Data release policy</h1>
 <p>As a community resource project, the 1000 Genomes Project publicly releases data on a regular basis. The sequencing centers release the raw reads to the SRA. The Data Processing Group, Analysis Group, and DCC release alignments, re-calibrated error rates, and SNP calls on individual samples. Data that passed quality filters are the most visible, but data that did not pass quality filters are also available. When the Structural Variation Group has developed its methods, the structural variant calls on individual samples will be released. Eventually the Project will release haplotypes and imputed variant calls. Data formats and analysis software developed by the Project are also made publicly available.</p>
</div>
<p><a name="SampleAvailability" id="SampleAvailability"></a></p>
<div class="enclosed">
 <h1 class="section_title">Sample Availability</h1>
 <p>All the samples studied by the 1000 Genomes Project are or will be available as DNA and cell lines to scientific investigators for research projects. Many of the samples are currently available from the non-profit <a href="http://ccr.coriell.org/sections/Collections/NHGRI/hapmap.aspx?PgId=266&amp;coll=GM">Coriell Institute for Medical Research</a>. Cell lines and DNA from additional populations will become available through 2011.</p>
</div>
<p><a name="DataUse" id="DataUse"></a></p>
<div class="enclosed">
 <h1 class="section_title">Use of the Project data, presentations and publications, and authorship</h1>
 <p>The data producers release the Project data quickly, prior to publication, in the expectation that they will be valuable for many researchers. In keeping with <a href="http://www.genome.gov/Pages/Research/WellcomeReport0303.pdf">Fort Lauderdale principles</a>, data users may use the data for many studies, but are expected to allow the data producers to make the first presentations and to publish the first paper with global analyses of the data.</p>
 <h3>Global analyses of Project data</h3>
 <p>The Project plans to publish global analyses of the sequence data and quality, SNPs, structural variants, STRs and <span>microsatellites</span>&nbsp;and haplotypes and LD patterns, population genetic phenomena such as population comparisons, mutation rates, and signals of selection, and functional annotations, as well as analyses of regions of general interest such as HLA. Talks, posters, and papers on all such analyses are to be published first by the 1000 Genomes Consortium, by approved presenters on behalf of the Project, with the Consortium as author. When the first major Project paper on these analyses is published, then researchers inside and outside the Project are free to present and publish using the Project data for these and other analyses.</p>
 <h3>Large-scale analyses of Project data</h3>
 <p>Groups within the Project may make presentations and publish papers on more extensive analyses of topics to be included in the main analysis presentations and papers, coincident with the main project analysis presentations and papers. The major points would be included in the main Project presentations and papers, but these additional presentations and papers allow more focused discussion of methods and results. The author list would include the Consortium.</p>
 <h3>Methods development using Project data</h3>
 <p>Researchers who have used small amounts of Project data (&lt;= one chromosome) may present methods development posters, talks, and papers that include these data prior to the first major Project paper, without needing Project approval or authorship, although the Project should be acknowledged. Methods presentations or papers on global analyses or analyses using large amounts of Project data, on topics that the Consortium plans to examine, would be similar to large-scale analyses of Project data: researchers within the Project may make presentations or submit papers at the same time as the main Project presentations and papers, and others could do so after the Project publishes the first major analysis paper.</p>
 <h3>Disease studies using Project data</h3>
 <p>Researchers may present and publish on use of Project data in specific chromosome regions (that are not of general interest) or as summaries (such as the total number of variants) for disease studies without Project approval, prior to the first major Project paper being published. The Project should not be listed as an author.</p>
 <h3>Population comparisons using Project data</h3>
 <p>Researchers may use Project data as controls or additional information for comparisons with their samples from people with particular diseases or from other populations, prior to the major Project paper being published, as long as the analyses that the Project plans to do are not included. These are not Project studies, and the Project should not be listed as an author.</p>
 <p>Researchers who have questions about whether they may make presentations or submit papers using Project data, or whether to include the 1000 Genomes Consortium as an author, may <a href="/contact">contact</a> Richard Durbin, David Altshuler, Gil McVean, Gonçalo Abecasis, or Lisa Brooks.</p>
</div>
  </div>

  
</div></div> <!-- /node-inner, /node -->          </div> <!-- /#main-content -->

          
		
		</div> <!-- // #content-inner -->
	  </div> <!-- //#content -->
	</div> <!-- //end #main -->

    
        <div id="sidebar-right" class="column sidebar region grid-3">
	    <div id="sidebar-right-inner">
          <div id="block-user-1" class="block block-user region-odd odd region-count-1 count-1"><div class="block-inner">

      <h2 class="title">Navigation</h2>
  
  <div class="content">
    <ul class="menu"><li class="leaf first last"><a href="/faq">Frequently Asked Questions</a></li>
</ul>  </div>

  
</div></div> <!-- /block-inner, /block -->
  <div id="block-block-3" class="block block-block region-even even region-count-2 count-2"><div class="block-inner">

      <h2 class="title">Links</h2>
  
  <div class="content">
    
<ul class="rightbar">
 <li id="megaphone"><a href="/announcements"><span>All project announcements</span></a></li>
 <li id="floppy"><a href="/formats"><span>Files and formats</span></a></li>
 <li id="tools"><a href="/tools"><span>Software tools</span></a></li>
 <li id="star"><a href="/sites/1000genomes.org/files/docs/nature09534.pdf"><span>Download the 1000 Genomes Pilot Paper</span></a></li>
</ul>
<ul class="rightbar">
 <li>&nbsp;</li>
 <li id="email"><a href="/contact"><span>Project Contacts</span></a></li>
</ul>
  </div>

  
</div></div> <!-- /block-inner, /block -->
      </div>
    </div> <!--//end #sidebar-right-inner -->
    

