---
layout: angularjs_partial
---

This study advances SV characterization by integrating linear and graph based calling methods to create the first population-scale catalogue of SVs using long-reads. 

1,019 samples from twentysix 1000 Genomes Project populations were sequenced to intermediate coverage using Oxford Nanopore Technologies (ONT). Linear and graph-based approaches for SV analysis were integrated via pangenome graph-augmentation, to uncover 167,291 sequence-resolved SVs, 164,571 of which were successfully phased.

This work is described in [Long-read sequencing and structural variant characterization in 1,019 samples from the 1000 Genomes Project by Schloissnig, Pani  et al](https://doi.org/10.1038/s41586-025-09290-7)

Data available includes:

- Alignments to [GRCh38](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1KG_ONT_VIENNA/hg38/)
- Alignments to [T2T-CHM13](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1KG_ONT_VIENNA/t2t/)
- Alignments to the [HPRC year-1 minigraph pangenome graph](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1KG_ONT_VIENNA/gaf/)
- A Giggles genotyped [call set of 167,291 SVs](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1KG_ONT_VIENNA/release/v1.1/giggles-genotyping/)
- A SHAPEIT5 phased integrated [call set of 164,571 SVs](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1KG_ONT_VIENNA/release/v1.1/shapeit5-phased-callset/)



Files can be browsed by sample, population and data type below. Files are available on the [IGSR FTP](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1KG_ONT_VIENNA/) and via the public Globus Collection EMBL-EBI Public Data, path /1000g/ftp/data_collections/1KG_ONT_VIENNA/
